### ISSUE TITLE
Add clear button to the search component

### ISSUE THREAD

**Issue by workshur** [provectus/kafka-ui#2686]
### Describe the solution you'd like
Search input should be clearable

### Additional context
Component: src/components/common/Search/Search.tsx

**Comment by detronetdip** [comment]
Can I work on this issue? can you please assign me this?

**Comment by Haarolean** [comment]
> Can I work on this issue? can you please assign me this?

sure :)

**Comment by detronetdip** [comment]
Hi @Haarolean I am very new to apache kafka can you please guide me hou to setup a local cluster so that I can test this UI and can work on this issue?

**Comment by detronetdip** [comment]
Hi @Haarolean  could you please help me?

**Comment by Haarolean** [comment]
@detronetdip please refer to:
[contributing guide](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md)
[frontend build guide](https://github.com/provectus/kafka-ui/blob/master/kafka-ui-react-app/README.md)
You'll have to run the compose: `docker-compose -f ./documentation/compose/kafka-ui.yaml up` and set `VITE_DEV_PROXY` to `http://localhost:8080`.

**Comment by Haarolean** [comment]
@detronetdip any luck?

**Comment by detronetdip** [comment]
@Haarolean I am working

**Comment by detronetdip** [comment]
@Haarolean  I followed you instruction, and on opening `localhost:3000` it is showing a blank screen

![Screenshot from 2022-10-20 21-57-15](https://user-images.githubusercontent.com/64276906/197005446-1751edbf-8427-4a3f-bf70-fd66d43b2a6d.png)

**Comment by Haarolean** [comment]
> @Haarolean I followed you instruction, and on opening `localhost:3000` it is showing a blank screen. I think I need your help again
> 
> ![Screenshot from 2022-10-20 21-57-15](https://user-images.githubusercontent.com/64276906/197005446-1751edbf-8427-4a3f-bf70-fd66d43b2a6d.png)

I suppose you could check the red error in your browser console as a start.

**Comment by detronetdip** [comment]
@workshur by saying search input should be clearable, did you mean there will be a button to clear the input field? and please guide me which search input you are taking about.

**Comment by Haarolean** [comment]
> @workshur by saying search input should be clearable, did you mean there will be a button to clear the input field? and please guide me which search input you are taking about.

Yes it's about having a button to clear the inout.
We're talking about all the search inputs in the app, there's a component class mentioned in the issue.

**Comment by detronetdip** [comment]
@Haarolean  I have run `docker-compose -f ./documentation/compose/kafka-ui.yaml up` command and have set `VITE_DEV_PROXY` to `http://localhost:8080`  in the `.env` file inside `kafka-ui-react-app` folder and then I visit `localhost:8080` in my browser and the finally I can see the UI but.  Is this the correct way? If it is then should I stop the docker containers and restart it again to view changes? Actually I am unable to find how to properly setup the development environment  and how to view my changes. Please guide me.

**Comment by Haarolean** [comment]
@detronetdip as it is stated in [frontend module readme](https://github.com/provectus/kafka-ui/tree/master/kafka-ui-react-app) if you develop the frontend part, you should run it via `pnpm start` and visit the link produced by this command. The running containers are used as a backend which you don't have to rebuild unless you do modifications to the backend side as well.

**Comment by detronetdip** [comment]
![Screenshot from 2022-10-30 22-12-34](https://user-images.githubusercontent.com/64276906/198890576-6699ea81-9fc4-4620-ade6-830bb865b086.png)
@Haarolean  It is showing this error.

**Comment by Haarolean** [comment]
It’s your ad blocker. On 31 Oct 2022, at 00:44, Ayondip Jana ***@***.***> wrote:﻿

@Haarolean  It is showing this error.

**Comment by detronetdip** [comment]
I have disabled the ad-blocker and now it is throwing a branch of errors

![Screenshot from 2022-10-30 22-17-25](https://user-images.githubusercontent.com/64276906/198890802-8f8309c7-a54c-4c92-8c49-ac2caa96647e.png)

**Comment by Haarolean** [comment]
> @Haarolean I have disabled the ad-blocker and now it is throwing a branch of errors
> 
> ![Screenshot from 2022-10-30 22-17-25](https://user-images.githubusercontent.com/64276906/198890802-8f8309c7-a54c-4c92-8c49-ac2caa96647e.png)

I honestly have no foggiest what you're doing wrong from this obscure screenshots. There's a decent guide which does work without any problems in 90% of cases. Please ensure that backend is accessible via localhost:3000/api/xxx, the proxy is set properly, etc.

**Comment by detronetdip** [comment]
Ok let try again from the very beginning.

**Comment by Haarolean** [comment]
@detronetdip any luck?

**Comment by Mano-Liaoyan** [comment]
Hi, I am looking forward to diving deep into the project, but I went into the same problem as @detronetdip did. After I followed the instruction on the readme page (stated below) and set the .env file as empty
<img width="811" alt="image" src="https://user-images.githubusercontent.com/43143420/226153059-4b62cfd7-af2a-41be-aa83-f171a498ff0f.png">
and I got a blank screen and the error in the browser console showed
<img width="423" alt="image" src="https://user-images.githubusercontent.com/43143420/226153082-27b368a3-d0db-4791-9e05-0af0fec6ab27.png">
<img width="507" alt="image" src="https://user-images.githubusercontent.com/43143420/226153100-acbe80d6-1b7d-4165-beb3-b298226023a5.png">
It seems that there are some cache problems with the front end when starting the project using pnpm. The backend works totally fine since I can access UI through the 8080 port. Do we actually need to configure the dev proxy even when using docker as the backend? If so, how to set it? The readme is not very clear with the API server. What is that and how to set it as a development environment? 
BTW, I am using WSL2 as my dev environment, hope this is not the cause of the problem
